### DISCUSSION TITLE
Align or merge the UIs of Navigation & Plan a route

### DISCUSSION THREAD

**Discussion by martin-frbg** [osmandapp/OsmAnd#17322] [Ideas]
"Plan a Route" seems to lack "Navigation"s ability to select existing markers or place names - but selecting points by pointing on the map gets tedious when the distance is large and/or the device is small. (worse if you hit the floating "current location" button by accident and get teleported back)

"Navigation" does not have this limitation, but it insists on including the current location in the route even when both starting point and destination are entered (and far away).
This makes it tedious to plan trips ahead when they will start from some future destination

**Comment by vshcherb** [comment]
Yes Navigation should  in future include Navigation by Markers (https://docs.osmand.net/docs/user/navigation/setup/markers-navigation) and Plan Route. It should rework how it displays multi-segment route and how much it insists on skipping intermediate point. For now navigation is ok for touch less navigation like a car for more freestyle navigation it doesn't fit
